From ec763f6cd3eae3ebe761abea4e73215ec6a53598 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 12 Jun 2018 17:37:12 +0200
Subject: [PATCH v2 1/1] x86: qemu: do not build car.o with start64.o

car.o can only be used with start.o, not with start64.o.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2
	rebase to current master
	shorten commit message
---
 arch/x86/cpu/qemu/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/x86/cpu/qemu/Makefile b/arch/x86/cpu/qemu/Makefile
index e5ea92545e3..b1daffd4a32 100644
--- a/arch/x86/cpu/qemu/Makefile
+++ b/arch/x86/cpu/qemu/Makefile
@@ -2,6 +2,9 @@
 #
 # Copyright (C) 2015, Bin Meng <bmeng.cn@gmail.com>
 
+ifndef CONFIG_$(SPL_)X86_64
+obj-y += car.o
+endif
-obj-y += car.o dram.o
+obj-y += dram.o
 obj-y += qemu.o
 obj-$(CONFIG_QFW) += cpu.o e820.o
-- 
2.17.1

